### PR TITLE
[EDU-4234] add revalidated status to edge cache doc

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Adds revalidated status to status definitions

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-4234]: https://aziontech.atlassian.net/browse/EDU-4234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx [hunk 1]
```diff
@@ -27,7 +27,9 @@ If origin servers experience downtime or become unavailable, Azion Edge Cache ca
...
+When the requested content is not available in the cache, Azion's Edge Cache module employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx:30]:
```suggestion
When the requested content isn't available in the cache, Azion's Edge Cache module employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx [hunk 2]
```diff
@@ -27,7 +27,9 @@ Se os servidores de origem estiverem inativos ou ficarem indisponíveis, o Edge
...
+Quando o conteúdo requisitado não está disponível em cache, o módulo **Edge Cache** da Azion emprega estratégias para minimizar o impacto no servidor de origem e otimizar a entrega de conteúdo. Uma dessas estratégias inclui manter conexões keep-alive com o servidor de origem sempre que possível, para reduzir a sobrecarga associada a handshakes TCP/IP frequentes. Isso é eficaz durante atualizações de cache ou em situações de um [cache MISS](#definicoes-de-status). 
+
+Além disso, a Azion implementa o controle de Thundering Herd para garantir que, diante de múltiplas requisições simultâneos para o mesmo objeto não armazenado em cache, apenas uma conexão seja estabelecida com o servidor de origem. Essa abordagem garante que cada um edge nodes da Azion solicite conteúdo ao servidor de origem apenas uma vez por MISS ou outros status não-HIT, otimizando assim a eficiência da rede e reduzindo a carga no servidor de origem.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx:32]:
```suggestion
Além disso, a Azion implementa o controle de Thundering Herd para garantir que, diante de múltiplas requisições simultâneos para o mesmo objeto não armazenado em cache, apenas uma conexão seja estabelecida com o servidor de origem. Essa abordagem garante que cada um edge nodes da Azion solicite conteúdo ao servidor de origem apenas uma vez por MISS ou outros status não-HIT, otimizando, assim, a eficiência da rede e reduzindo a carga no servidor de origem.
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx (hunk 1 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx:30]:
```suggestion
When the requested content isn't available in the cache, the Azion Edge Cache module employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx [hunk 3]
```diff
@@ -48,6 +50,7 @@ Saiba mais sobre como automatizar seu navegador para obter outros dados de cache
 | **EXPIRED** | O conteúdo em cache expirou o TTL definido no edge. Quando a origem responde, o conteúdo em cache é atualizado para futuras requisições |
 | **STALE** | Ao optar por servir [stale cache](#stale-cache), se a origem falhar em responder e o cache edge expirou, uma versão obsoleta do conteúdo é servida |
 | **UPDATING** | O conteúdo em cache expirou e o stale cache está sendo servido, mas o conteúdo será atualizado para futuras requisições |
+| **REVALIDATED** | O conteúdo em cache expirou ou foi previamente servido como stale, mas não foi modificado desde o momento em que foi registrado. Se o cache ainda for válido, a edge retorna este status e não irá requisitar o conteúdo completo da origem |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx:53]:
```suggestion
| **REVALIDATED** | O conteúdo em cache expirou ou foi previamente servido como stale, mas não foi modificado desde o momento em que foi registrado. Se o cache ainda for válido, o edge retorna este status e não irá requisitar o conteúdo completo da origem |
```

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx (hunk 2 above)
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx]:
```suggestion
Além disso, a Azion implementa o controle de Thundering Herd para garantir que, diante de múltiplas requisições simultâneas para o mesmo objeto não armazenado em cache, apenas uma conexão seja estabelecida com o servidor de origem. Essa abordagem garante que cada um dos edge nodes da Azion solicite conteúdo ao servidor de origem apenas uma vez por MISS ou outros status não-HIT, otimizando assim a eficiência da rede e reduzindo a carga no servidor de origem.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx [hunk 4]
```diff
@@ -48,6 +50,7 @@ Find out more about how to automate your browser to obtain other cache data in t
 | **EXPIRED** | The cached content has expired the TTL set at the edge. When the origin responds, the cached content is updated for future requests |
 | **STALE** | When choosing to serve [stale cache](#stale-cache), if the origin fails to respond and the edge cache has expired, a stale version of the content is served |
 | **UPDATING** | The cached content has expired and a stale cache is being served, but the content is being updated in the origin |
+| **REVALIDATED** | The cached content has expired or has been previously served as stale, but hasn't been modified since the time it was recorded. If the cache is still valid, the edge returns this status and won't request the full content from the origin |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx:53]:
```suggestion
| **REVALIDATED** | The cached content is checked against the origin using conditional headers. If it's still up-to-date, the resource is not retransmitted. |
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx [hunk 5]
```diff
@@ -27,7 +27,9 @@ If origin servers experience downtime or become unavailable, Azion Edge Cache ca
...
+When the requested content isn't available in the cache, the Azion Edge Cache module employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx:30]:
```suggestion
When the requested content isn't available in the cache, the Azion Edge Cache add-on employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).
```

Já é pra mudar?

**Pull request merged**